### DISCUSSION TITLE
supabase.auth.token in Local Storage

### DISCUSSION THREAD

**Discussion by mwhitlaw** [supabase/supabase#3640] [Questions]
I see that supabase is storing the supabase.auth.token in Local Storage. Is this secure? Could any js library included in my app read that data and relay it to some other place where it could be used to access my app?

**Comment by hf** [comment]
Hey, there's no way to "secure" content stored in your browser from other libraries you include in your app. Please make sure you review any libraries you use for vulnerabilities if this is a concern for you. You can use [`npm audit`](https://docs.npmjs.com/cli/v8/commands/npm-audit) or equivalents to constantly monitor your dependencies for vulnerabilities, and there are a few products out there (GitHub Dependabot) that do this for you too.
